converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java

```diff
@@ -127,17 +127,6 @@ private void setTaxProviderTaxIdOnNode(Element element, Node thisNode) {
 				consumer, Filters.attribute(), true);
 	}
 
-	/**
-	 * Continues decoding the elements that are children of Clinical Document.
-	 *
-	 * @param element Xml fragment being parsed.
-	 * @param thisNode The output internal representation of the document
-	 */
-	private void processComponentElement(Element element, Node thisNode) {
-		Consumer<Element> consumer = p -> this.decode(p, thisNode);
-		setOnNode(element, getXpath("components"), consumer, Filters.element(), false);
```

Comment:
This is no longer necessary? Not necessary to begin with? If so then we should evaluate the necessity of the components key in the path-correlation.json file. Did a quick search and I don't see any other usages.

Reply:
I removed this because this was an example of a decoder doing recursion (notice the call on line 137 to `this.decode(...)`).  Since I consolidated all the recursion logic to `QrdaDecoderEngine`, this method is no longer necessary.